src/settings/settings.go

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 type Settings struct {
+	Proto string `envconfig:"PROTO" default:"tcp"`
```

Comment:
Nit: how about using `Protocol` instead of `Proto`?  In our context, `Proto` could be mistaken with `ProtoBuf`.

Reply:
Or maybe a more straighforward `DualStack` or `EnableIPV6` ?

Reply:
this align to `reuseport.Listen(proto, addr string) (l net.Listener, err error)`